app/views/rooms/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with model: [room.space, room], data: { controller: "room-form" }, local: true do |room_form| %>
+<%= form_with model: room.location, data: { controller: "room-form" } do |room_form| %>
```

Comment:
why `local: true` no longer needed?

Reply:
It was needed back when we had both `turbo` and `rails-ujs` running in the project.